tests/dir.sdkbuild.props

```diff
@@ -7,7 +7,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <RuntimeFrameworkVersion>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>$(MicrosoftNETCorePlatformsPackageVersion)</RuntimeFrameworkVersion>
```

Comment:
You should know that this doesn't map to the version of Microsoft.NETCore.App if that is what you are trying to get here. It does it a lot of cases but not in all cases.

Reply:
Oh, interesting. Is there a better value to use?

Reply:
It doesn't look like you guys consume Microsoft.NETCore.App directly in coreclr. If it is a true dependency then you guys should start consuming it like corefx does https://github.com/dotnet/corefx/blob/master/dependencies.props#L42.